client.go

```diff
@@ -34,6 +34,7 @@ type Client struct {
 // NewClient returns a new Isilon client struct initialized from the environment.
 func NewClient(ctx context.Context) (*Client, error) {
 	insecure, _ := strconv.ParseBool(os.Getenv("GOISILON_INSECURE"))
+	isiAuthType, _ := strconv.Atoi(os.Getenv("GOISILON_ISIAUTHTYPE"))
```

Comment:
Please log the error appropriately. If there were to be a typo, it would always default to 0 which may not be the case. Always good to log errors. Please log the error from previous line as well